netlogo-core/src/main/fileformat/NLogoLabFormat.scala

```diff
@@ -16,7 +16,7 @@ trait LabFormat[A <: ModelFormat[Array[String], A]]
 
   def componentName = "org.nlogo.modelsection.behaviorspace"
 
-  val loader = new LabLoader(literalParser, editNames)
+  val loader = new LabLoader(literalParser)
```

Comment:
Similar to the previous comment - is `editNames` being made available in a different way?